js/binary/reader_test.js

```diff
@@ -42,6 +42,8 @@
  */
 
 goog.require('goog.testing.asserts');
+
+// CommonJS-LoadFromFile: google_protobuf
```

Comment:
Ah, it's because for CommonJS I don't currently test 100% of the `*_test.js` files. Several of the `binary/*_test.js` files access internal symbols from the library (or extra symbols from Google Closure) and making all of that work with CommonJS seemed to cross a line of being more work than it was worth.

But it's a good catch -- if we're not testing `reader_test.js`, there is no point in adding this line to it. So I removed this line from `reader_test.js` and `utils_test.js`.

I tried to add a comment to `commonjs/jasmine.json` to make it more clear that we're leaving out some files. But unfortunately it is JSON and cannot have comments.